airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionTransformationTab/DbtCloudTransformationsCard.tsx

```diff
@@ -0,0 +1,181 @@
+  };
+
+  return (
+    <Formik // TODO extract to parent component, see if that helps with input focus issues
+      onSubmit={onSubmit}
+      initialValues={{ jobs: dbtCloudJobs }}
+      validationSchema={dbtCloudJobListSchema}
+      render={({ values, isValid, dirty }) => {
+        return (
+          <Form className={styles.jobListForm}>
+            <FormChangeTracker changed={dirty} />
+            <FieldArray
+              name="jobs"
+              render={({ remove, push }) => {
+                return (
+                  <Card
+                    title={
+                      <span className={styles.jobListTitle}>
+                        Transformations
+                        {hasDbtIntegration && (
+                          <Button
+                            variant="secondary"
+                            onClick={() => push({ account: "", job: "" })}
+                            icon={<FontAwesomeIcon icon={faPlus} />}
+                          >
+                            Add transformation
+                          </Button>
+                        )}
+                      </span>
+                    }
+                  >
+                    {hasDbtIntegration ? (
+                      <DbtJobsList jobs={values.jobs} remove={remove} isValid={isValid} dirty={dirty} />
+                    ) : (
+                      <NoDbtIntegration className={styles.jobListContainer} />
+                    )}
+                  </Card>
+                );
+              }}
+            />
+          </Form>
+        );
+      }}
+    />
+  );
+};
+
+const DbtJobsList = ({
+  jobs,
+  remove,
+  isValid,
+  dirty,
+}: {
+  jobs: DbtCloudJob[];
+  remove: (i: number) => void;
+  isValid: boolean;
+  dirty: boolean;
+}) => (
+  <div className={classNames(styles.jobListContainer, styles.emptyListContent)}>
+    <p className={styles.contextExplanation}>After an Airbyte sync job has completed, the following jobs will run</p>
+    {jobs.length ? (
+      jobs.map((_j, i) => <JobsListItem key={i} jobIndex={i} removeJob={() => remove(i)} />)
+    ) : (
+      <>
+        <img src="/images/octavia/worker.png" alt="An octopus wearing a hard hat, tools at the ready" />
+        No transformations
+      </>
+    )}
+    <div className={styles.jobListButtonGroup}>
+      <Button className={styles.jobListButton} type="reset" variant="secondary">
+        Cancel
+      </Button>
+      <Button className={styles.jobListButton} type="submit" variant="primary" disabled={!dirty || !isValid}>
+        Save changes
+      </Button>
+    </div>
+  </div>
+);
+
+// TODO give feedback on validation errors (red outline and validation message)
+const JobsListItem = ({ jobIndex, removeJob }: { jobIndex: number; removeJob: () => void }) => {
+  return (
+    <Card className={styles.jobListItem}>
+      <div className={styles.jobListItemIntegrationName}>
+        <img src="/images/external/dbt-bit_tm.png" alt="dbt logo" />
```

Comment:
I think we should try to give this image a bit of margin to the right, it's currently attached a lot to the text.